packages/frontend/assets/package.json

```diff
@@ -9,12 +9,10 @@
   },
   "devDependencies": {
     "@shared/eslint-config": "*",
-    "@shared/tsconfig": "*"
-  },
-  "peerDependencies": {
-    "expo-image": "*",
-    "react": "*",
-    "react-native-svg": "*"
+    "@shared/tsconfig": "*",
+    "expo-image": "~1.10.6",
+    "react": "18.2.0",
+    "react-native-svg": "^15.1.0"
```

Comment:
[question] Why are these only dev dependencies?

Reply:
Because in `assets` these packages are only used to generate types I believe - cc @RSAMUEL12 @Francisco-Costa98 

The idea is that typescript is converted to javascript code in development (and not needed in the real bundle?)

Reply:
I think this should be fine but might have to be tested if changing it to a devDependency affects the usage of the asset files/components

Reply:
was working for me 👍